packages/ddl-shim/src/upgrade.ts

```diff
@@ -704,8 +707,11 @@ class DDLUpgrade {
     }
 
     writeProperties(): DDL2.IProperties | undefined {
-        //  TODO:  Upgrade v1.x.x dermatologies  ---
-        return undefined;
+        return {
+            name: "@hpcc-js/marshaller", // TODO - should PKG_NAME be used here?
+            version: PKG_VERSION,
+            buildVersion: BUILD_VERSION
+        };
```

Comment:
The properties its supposed to write here, would be the Grid or Dock Panel properties.

Reply:
Are you saying that this should be...
...for Grid:
```json
{
    name: "@hpcc-js/layout",
    version: layout_PKG_VERSION,
    buildVersion: layout_BUILD_VERSION
}
```
...for DockPanel:
```json
{
    name: "@hpcc-js/phosphor",
    version: phosphor_PKG_VERSION,
    buildVersion: phosphor_BUILD_VERSION
}
```

Reply:
Nevermind, I see what you mean, and yes, the `Grid` properties are written there within `packages/ddl-shim/src/dermatology.ts`